### PR TITLE
Make data and locale optional in multilingual LG

### PR DESCRIPTION
**Pull request by Danieladu**
close: #3931

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.LanguageGeneration/MultiLanguageLG.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ public MultiLanguageLG(Dictionary<string, Templates> templatesPerLocale, string
         /// <param name="data">Scope data.</param>
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.LanguageGeneration/MultiLanguageLG.cs:59]: >data [](start = 25, length = 4)

Why not call this scope like other APIs like Templates.Evaluate?

Comment (Danieladu) [reply libraries/Microsoft.Bot.Builder.LanguageGeneration/MultiLanguageLG.cs:59]: At the very beginning MultilanguageLG implements ILanguageGenerator interface which has the method "generate". But now there is no such dependency, we can consider changing the name.
@boydc2014 any thougths?

Comment (boydc2014) [reply libraries/Microsoft.Bot.Builder.LanguageGeneration/MultiLanguageLG.cs:59]: I don't have naming preference, i think keep the name "Generate" is good to me, we don't have to introduce name change here, @vishwacsena  what do you think?

Comment (vishwacsena) [reply libraries/Microsoft.Bot.Builder.LanguageGeneration/MultiLanguageLG.cs:59]: Yes. Do not believe we need a method name change here.

**Pull request merged**